CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## XX.XX.XX - 2023-XX-XX
 
+### PaymentSheet
+* [ADDED] PaymentSheet now supports the following payment methods for SetupIntents and PaymentIntents with setup for future usage:
+  * [7282](https://github.com/stripe/stripe-android/pull/7282) Boleto
```

Comment:
Nit: Should make this a single line if we don’t end up adding the same support for more payment methods in this release.